lib/msal-browser/src/operatingcontext/MetaOSOperatingContext.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { BaseOperatingContext } from "./BaseOperatingContext";
+
+export class MetaOSOperatingContext extends BaseOperatingContext {
+    
+    /*
+     * TODO: Once we have determine the bundling code return here to specify the name of the bundle
+     * containing the implementation for this operating context
+     */
+    static readonly MODULE_NAME: string = "";
+
+    /**
+     * Unique identifier for the operating context
+     */
+    static readonly ID: string = "MetaOSOperatingContext";
+
+    /**
+     * Return the module name.  Intended for use with import() to enable dynamic import
+     * of the implementation associated with this operating context
+     * @returns 
+     */
+    getModuleName(): string {
+        return MetaOSOperatingContext.MODULE_NAME;
+    }
+    /**
+     * Returns the unique identifier for this operating context
+     * @returns string
+     */
+    getId(): string {
```

Comment:
Implementations for getModuleName and getId look the same in both Operating Contexts. Any reason they're not defined in the Base Context?

Reply:
That makes sense.  I'll take a look

Reply:
Let me know if you feel strongly about this

Reply:
These are not actually the same though...

Reply:
Ah I see, this can't be moved because the Id and ModuleName are static, is that right? Do they have to be static? If so feel free to resolve.